crates/pathfinder/src/sierra.rs

```diff
@@ -184,4 +235,28 @@ mod tests {
             compile_to_casm(&contract_definition, &StarknetVersion::new(0, 11, 1)).unwrap();
         }
     }
+
+    mod starknet_v0_11_2_onwards {
+        // TODO: replace test class with v0.11.2 compiler class.
```

Comment:
Do we wait for a real v0.11.2 class on integration? Or do we just compile our own locally?

Reply:
I opted to just leave it as a todo.